role-lambda.tf

```diff
@@ -0,0 +1,76 @@
+locals {
+  count = "${var.lambda_role_name != "" ? 0 : 1}"
```

Comment:
Can we get a comment about what `count` "means" here? Like what causes it to be zero and what happens then?

Reply:
Is this supposed `local.lambda_role_name`?

Reply:
This should be `var` instead of local since `local` will always have a value.

I added a comment about `count` that I hope is helpful–basically, if we provide a custom role, we don't create the default role or the CF stack, but _do_ create and attach the policies to the custom role.